packages/css-library/dist/tokens/css/variables.css

```diff
@@ -192,116 +193,45 @@
   --units-neg-9: -4.5rem;
   --units-neg-10: -5rem;
   --units-neg-15: -7.5rem;
-  --uswds-system-color-blue-5: #eff6fb;
-  --uswds-system-color-blue-10: #d9e8f6;
-  --uswds-system-color-blue-20: #aacdec;
-  --uswds-system-color-blue-30: #73b3e7;
-  --uswds-system-color-blue-40: #4f97d1;
-  --uswds-system-color-blue-50: #2378c3;
-  --uswds-system-color-blue-60: #2c608a;
-  --uswds-system-color-blue-70: #274863;
-  --uswds-system-color-blue-80: #1f303e;
-  --uswds-system-color-blue-90: #11181d;
-  --uswds-system-color-blue-vivid-30: #58b4ff;
-  --uswds-system-color-blue-vivid-50: #0076d6;
-  --uswds-system-color-blue-vivid-60: #005ea2;
-  --uswds-system-color-blue-vivid-70: #0b4778;
-  --uswds-system-color-blue-vivid-80: #112f4e;
-  --uswds-system-color-blue-cool-60: #2e6276;
-  --uswds-system-color-blue-cool-vivid-5: #e1f3f8;
-  --uswds-system-color-blue-cool-vivid-20: #97d4ea;
-  --uswds-system-color-blue-cool-vivid-40: #28a0cb;
-  --uswds-system-color-blue-cool-vivid-60: #07648d;
-  --uswds-system-color-blue-warm-5: #ecf1f7;
-  --uswds-system-color-blue-warm-10: #e1e7f1;
-  --uswds-system-color-blue-warm-30: #98afd2;
-  --uswds-system-color-blue-warm-50: #4a77b4;
-  --uswds-system-color-blue-warm-60: #345d96;
-  --uswds-system-color-blue-warm-80: #252f3e;
-  --uswds-system-color-blue-warm-vivid-60: #0050d8;
-  --uswds-system-color-blue-warm-vivid-70: #1a4480;
-  --uswds-system-color-blue-warm-vivid-80: #162e51;
-  --uswds-system-color-cyan-5: #e7f6f8;
-  --uswds-system-color-cyan-20: #99deea;
-  --uswds-system-color-cyan-vivid-30: #00bde3;
-  --uswds-system-color-cyan-vivid-40: #009ec1;
-  --uswds-system-color-gold-50: #8e704f;
-  --uswds-system-color-gold-vivid-5: #fef0c8;
-  --uswds-system-color-gold-vivid-10: #ffe396;
-  --uswds-system-color-gold-vivid-20: #ffbe2e;
-  --uswds-system-color-gold-vivid-30: #e5a000;
-  --uswds-system-color-gold-vivid-40: #c2850c;
-  --uswds-system-color-gold-vivid-50: #936f38;
-  --uswds-system-color-gray-1: #fcfcfc;
-  --uswds-system-color-gray-2: #f9f9f9;
-  --uswds-system-color-gray-3: #f6f6f6;
-  --uswds-system-color-gray-4: #f3f3f3;
-  --uswds-system-color-gray-5: #f0f0f0;
-  --uswds-system-color-gray-10: #e6e6e6;
-  --uswds-system-color-gray-20: #c9c9c9;
-  --uswds-system-color-gray-30: #adadad;
-  --uswds-system-color-gray-40: #919191;
-  --uswds-system-color-gray-50: #757575;
-  --uswds-system-color-gray-60: #5c5c5c;
-  --uswds-system-color-gray-70: #454545;
-  --uswds-system-color-gray-80: #2e2e2e;
-  --uswds-system-color-gray-90: #1b1b1b;
-  --uswds-system-color-gray-100: #000000;
-  --uswds-system-color-gray-cool-1: #fbfcfd;
-  --uswds-system-color-gray-cool-2: #f7f9fa;
-  --uswds-system-color-gray-cool-3: #f5f6f7;
-  --uswds-system-color-gray-cool-4: #f1f3f6;
-  --uswds-system-color-gray-cool-5: #edeff0;
-  --uswds-system-color-gray-cool-10: #dfe1e2;
```

Comment:
I found instances of a few of these uswds-system-color variables being used in vets-website.
![Screenshot 2025-01-29 at 10 36 21 AM](https://github.com/user-attachments/assets/8601fc31-d736-4282-a362-52f40bb19217)

Reply:
Looks to be about 9-10 instances right now.

Reply:
@Andrew565 - yeah I'm aware of those. Created a ticket to replace these yesterday - https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/3756 - when we do our next css-library release.

Reply:
Nice!